src/dbus.c

```diff
@@ -62,7 +69,24 @@ static const char *introspection_xml =
     "            <arg name=\"id\"         type=\"u\"/>"
     "            <arg name=\"action_key\" type=\"s\"/>"
     "        </signal>"
-    "   </interface>"
+    "    </interface>"
+    "    <interface name=\""DUNST_IFAC"\">"
+
+    "        <method name=\"ContextMenuCall\"       />"
+// TODO: add an optional parmater definining the action of notification number X to invoke
```

Comment:
i skipped reading through this file, because there are too much `TODOs` and `FIXMEs`.
but testing wise, everything works perfectly fine for me.